apps/qubit/modules/settings/actions/globalAction.class.php

```diff
@@ -213,6 +219,43 @@ protected function updateGlobalSettings()
       }
     }
 
+    // Identifier mask enabled
+    if (null !== $identifierMaskEnabled = $thisForm->getValue('identifier_mask_enabled'))
+    {
+      if (null === $setting = QubitSetting::getByName('identifier_mask_enabled'))
+      {
+        $setting = new QubitSetting;
```

Comment:
Why do you create this one in here an not the others? We use to do this when the settings are not added via migration script, but you're adding them.